src/js/edu-benefits/definitions/toursOfDuty.js

```diff
@@ -0,0 +1,64 @@
+import * as dateRange from '../../common/schemaform/definitions/dateRange';
+import ServicePeriodView from '../components/ServicePeriodView';
+
+/**
+ * Returns schema for toursOfDuty based on the property names passed to it.
+ * This is because some forms don't need all the information others do.
+ *
+ * Note: The order in which the names are in the array will affect the order
+ *  they will appear in the form.
+ */
+export function schema(propNames = ['serviceBranch', 'dateRange']) {
+  const todSchema = {
+    type: 'array',
+    items: {
+      type: 'object',
+      properties: {}
```

Comment:
Another way to write this would be to move the properties object above this and do:

```js
properties: _.pick(propNames, possibleProperties)
```